src/components/Navbar/Navbar.scss

```diff
@@ -1,18 +1,147 @@
-@import './../../styles/fonts.scss';
-
-.nav-text {
-  font-family: $poppins;
-  font-weight: 300;
-  font-size: 17px;
-  line-height: 25px;
-  text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.03);
+@import '../../styles/colors.scss';
+
+.nav {
+  height: 10rem;
+  background-color: $white;
+  padding: 0 1.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  filter: drop-shadow(0px -4px 10px rgba(0, 0, 0, 0.25));
+
+  &--container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &--logo {
+    height: 7rem;
+    width: 7rem;
+    margin-left: -.5rem;
+  }
+
+  &--title {
+    font-weight: 700;
+    color: $dark-grey;
+    font-size: 1.5rem;
+    margin: 0;
+    padding: 0;
+
+    &__sm {
+      color: $black;
+      font-size: .8rem;
+      margin: 0;
+      padding: 0;
+    }
+
+    @media (max-width: 1300px) {
+      font-size: 1.3rem;
+    }
+  }
+
+  &--items {
+    width: 55%;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &--titles {
+    margin-left: 1rem;
+    display: flex;
+    flex-direction: column;
+    font-size: 17px;
+    line-height: 25px;
+    text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.03);
+  }
+
+  &--link {
+    color: $black;
+    text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.03);
+    border: 0;
+    cursor: pointer;
+    padding: 2rem .5rem;
+
+    &:hover {
+      color: $dark-purple;
+    }
+
+    &__txt {
+      display: flex;
+      color: $black;
+      align-items: center;
+      justify-content: center;
+      font-size: 1.1rem;
+
+      &:hover {
+        color: $dark-purple;
+        text-shadow: 0 0 .9px $dark-purple, 0 0 .9px $dark-purple, 0 0 .9px $dark-purple;
+      }
+
+      @media (max-width: 1300px) {
+        font-size: .9rem;
+      }
+    }
+  }
+
+  &--dropdown {
+    position: absolute;
+    z-index: 10;
+    top: 7rem;
+    background-color: $white;
+    border: 1px solid rgba(0, 0, 0, 0.09);
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    overflow: hidden;
+    padding: .5rem;
+
+    &__link {
+      color: $black;
+      display: flex;
+      align-items: center;
+      padding: .5rem;
+
+      &--txt {
+        font-size: 1rem;
+      }
+
+      &:hover {
+        color: $dark-purple;
+        text-shadow: 0 0 .9px $dark-purple, 0 0 .9px $dark-purple, 0 0 .9px $dark-purple;
+      }
+    }
+
+    &__links {
+      list-style-type: none;
+      padding: 0;
+    }
+  }
 }
 
-.button-text {
-  font-family: $poppins;
-  font-weight: 600;
-  font-size: 15px;
-  line-height: 22px;
-  letter-spacing: 0.03em;
-  text-transform: uppercase;
+.donate--link {
```

Comment:
This should be under `.nav` since thats its parent element - maybe a good name would be `.nav--donateLink`